.github/workflows/test.yaml

```diff
@@ -146,6 +146,50 @@ jobs:
         name: coverage
         path: coverage.enterprisepostgres.out
 
+  integration-tests-enterprise-postgres-nightly:
+    environment: "Configure ci"
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: Kong/kong-license@master
+      id: license
+      with:
+        # PULP_PASSWORD secret is set in "Configure ci" environment
+        password: ${{ secrets.PULP_PASSWORD }}
+
+    - name: setup golang
+      uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
+
+    - name: cache go modules
+      uses: actions/cache@v3
+      with:
+        path: ~/go/pkg/mod
+        key: ${{ runner.os }}-build-codegen-${{ hashFiles('**/go.sum') }}
+        restore-keys: |
+          ${{ runner.os }}-build-codegen-
+
+    - name: checkout repository
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: run integration tests
+      run: make test.integration.enterprise.postgres
+      env:
+        KONG_LICENSE_DATA: ${{ steps.license.outputs.license }}
+        TEST_KONG_IMAGE: "kong/kong-gateway-internal"
```

Comment:
It's the internal version of https://hub.docker.com/r/kong/kong-gateway/, i.e. the image that contains Enterprise stuff. Our permissions on Docker Hub usually only extend access to internal resources to various bot users, so it looks like a 404 on a  personal account.

Reply:
Ok got it, in that case perhaps we could add some documentation around it (I don't care if it's going to be here or elsewhere in the repo)?

I'm just trying to prevent folks - reading this in the future - scratching their heads, trying to figure out what's that.